api/eds.proto

```diff
@@ -191,6 +191,10 @@ message UpstreamLocalityStats {
 
   // Stats for multi-dimensional load balancing.
   repeated EndpointLoadMetricStats load_metric_stats = 5;
+
+  // [#not-implemented-hide:] The priority of the endpoint group these metrics
+  // were collected from.
+  google.protobuf.UInt32Value priority = 6;
```

Comment:
I would make this just `uint32`, since priority is always implied and hence should always be set.

Reply:
but if they don't set it, it will default to 0. I guess that is OK? We could also leave as WKT and mark as required?

Reply:
0 seems fine by me, since that's the default priority level.  I'm good either way, really, so I'll go with whatever you two prefer.

Reply:
This is something that is set by Envoy itself, rather than the user. So, we will always set it. OTOH, this API is nominally proxy agnostic, so we should think about other proxies that might generate load reports one day. However, the rest of the API implies 0 as @alyssawilk points out, so unwrapped seems best here.

Reply:
If we're worried about other proxies I think consistency with LBEndpoint makes the most sense.  so uint32_t for both or neither?

Reply:
For both, let's do it.